packages/falso/src/lib/random-float.ts

```diff
@@ -1,20 +1,11 @@
-import { rand } from './core';
+type Options = {
+  min: number;
+  max: number;
+  fractions?: number;
+};
 
-export function randomFloat() {
-  return rand([
-    91612.37, 14694.67, 23147.79, 49753.73, 60800.67, 89789.58, 5284.01,
-    50046.23, 59255.64, 53789.24, 87018.38, 89856.4, 2513.31, 80010.34,
-    37942.48, 58102.02, 30809.14, 63033.5, 19296.51, 69288.18, 40277.62,
-    17316.45, 78581.72, 13896.67, 174.93, 63545.02, 7445.72, 96733.41, 24987.66,
-    3177.45, 90446.6, 39452.89, 44866.63, 42942.59, 85474.52, 41694.63,
-    54963.36, 53413.44, 6931.08, 85866.74, 93443.29, 31032.71, 27862.74,
-    57176.01, 41396.54, 2189.21, 98733.61, 34013.12, 15563.95, 78388.54,
-    33880.8, 9421.84, 71111.08, 4295.1, 58694.31, 97777.87, 70603.09, 16337,
-    8699.48, 23789.87, 6844.22, 15893.96, 18897.09, 84017.57, 7102.99, 22060.28,
-    51955.51, 91389.33, 53076.53, 72607.07, 96320.23, 97593.35, 47203.98,
-    69409.58, 75603.57, 44673.23, 10593.59, 54691.27, 87488.8, 34331.1,
-    74486.94, 98411.08, 15360.95, 34152.43, 4361.13, 53308.27, 63902.05,
-    56419.32, 79839.49, 88826.74, 56163.71, 96314.6, 54707.54, 6767.67,
-    30910.92, 70763.65, 43188.82, 63343.58, 84665.45, 3176.64,
-  ]);
+export function randomFloat(
+  { min, max, fractions }: Options = { min: 1.0, max: 999.99 }
```

Comment:
Allow passing only max/min

Reply:
Remove the default values? Or remove the fractions digits?

Reply:
allow the following:

```ts
randomFloat({ min: 4 })
randomFloat({ max: 400 })
randomFloat({ fraction: 3 })
randomFloat({ fraction: 3, min: 3, max: 6 })
```